lemur/certificates/views.py

```diff
@@ -388,6 +388,7 @@ def get(self):
         return service.render(args)
 
     @validate_schema(certificate_input_schema, certificate_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
for some of our users, the self service nature of lemur is an important feature. We'd likely want this gated with a documented configuration property e.g, ADMIN_ONLY_CERTIFICATE_CREATION

Reply:
i agree with having a new configuration property. i noticed that when users are initially created a role is not required. so the current updates would restrict users that are not assigned a role.

for the new configuration property, this would allow the `operator` and `admin` role to access the endpoints and enforce the `read-only` role. instead of `ADMIN_ONLY_*`, what if we added a property called `LEMUR_STRICT_ROLE_ENFORCEMENT=(True|False)` (or something similar). 

if this is set it will activate/enforce the default system roles and require admins to assign a role when creating users. it will default to `False` to ensure backward compatibly.